Source/Core/AudioCommon/AlsaSoundStream.cpp

```diff
@@ -51,8 +57,11 @@ void AlsaSound::SoundLoop()
 	Common::SetCurrentThreadName("Audio thread - alsa");
 	while (m_thread_status.load() == ALSAThreadStatus::RUNNING)
 	{
+		std::unique_lock<std::mutex> lock(cv_m);
+		cv.wait(lock, [this]{return !m_muted || m_thread_status.load() != ALSAThreadStatus::RUNNING;});
+
 		m_mixer->Mix(reinterpret_cast<short *>(mix_buffer), frames_to_deliver);
-		int rc = m_muted ? 1337 : snd_pcm_writei(handle, mix_buffer, frames_to_deliver);
+		int rc = snd_pcm_writei(handle, mix_buffer, frames_to_deliver);
```

Comment:
I'm back working on this issue.

I'm actually confused by the semantic of the Clear() method. What is it actually meant to do? Clear (e.g zero) the buffer? Pause playback?